includes/classes/Pay/Gateway/Abstract.php

```diff
@@ -42,13 +43,11 @@ public static function getId()
     public function getIcon()
     {
         $size = get_option('paynl_logo_size');
-
         if ($size) {
-            return 'https://static.pay.nl/payment_profiles/' . $size . '/' . $this->getOptionId() . '.png';
+          return PAYNL_PLUGIN_URL . '/assets/logos/' . $this->get_option('brand_id') . '.png';
```

Comment:
De eerste slash voor assets kan er af. 
De code is nu ook wat vreemd, want wat doet size nu nog?
Kunnen de images in de checkout wat groter? Ik vind het prima als we $size hiervoor gebruiken.